xml/ha_maintenance.xml

```diff
@@ -117,7 +161,7 @@ Node <replaceable>&node2;</replaceable>: standby
    </step>
    <step>
     <para>
-     Check if the &pace; service has started:
+     Check if the cluster services has started:
```

Comment:
```suggestion
     Check if the &pace; cluster service has started:
```

Reply:
Is the singular OK? Not sure...

Reply:
That was on purpose. See my comment in https://github.com/SUSE/doc-sleha/pull/132/files#r415655616. It's plural and I would avoid using Pacemaker here. So "cluster services" should be ok

Reply:
In that case:

"... Check if the cluster services **have** started:"